src/bot/modules/stats.ts

```diff
@@ -43,33 +66,46 @@ export default class Stats {
           Proposées: **${corrections.length}**
           En attente: **${corrections.filter((s) => !s.refused && !s.merged).length}**
           Acceptées: **${corrections.filter((s) => s.merged).length}**
+          Votes: **${reaction.up} ${corrections_votes.filter((v) => v.type === VoteType.UP).length} ${reaction.down} ${
+          suggest_votes.filter((v) => v.type === VoteType.DOWN).length
+        }**
         `,
         inline: true
       }
     ];
 
     if (member.roles.cache.has(godfatherRoleId)) {
-      const approvals = await prisma.approval.findMany({
-        where: { user_id: member.id },
-        include: { proposal: true }
-      });
-      const disapprovals = await prisma.disapproval.findMany({
-        where: { user_id: member.id },
-        include: { proposal: true }
-      });
-
-      const totalDecisionsCount = approvals.length + disapprovals.length;
-      const suggestsDecisionsCount = [...approvals, ...disapprovals].filter(
+      const [approvals, disapprovals] = await Promise.all([
+        prisma.approval.findMany({
+          where: { user_id: member.id },
+          include: { proposal: true }
```

Comment:
Peut-être qu'on a pas besoin de récupérer toute la proposal ?
```suggestion
          include: { proposal: { select: { type: true } } }
```